test/jest/__mock__/stripesCore.mock.js

```diff
@@ -71,6 +71,8 @@ jest.mock('@folio/stripes/core', () => {
       return <Component {...rest} mutator={fakeMutator} resources={fakeResources} stripes={fakeStripes} />;
     },
 
+    useOkapiKy: jest.fn(),
```

Comment:
maybe it's better to mock more complex? like return a promise or something

Reply:
it returns `ky`, and you always need to define how it should behave (I mean its methods and return values)